src/midje/emission/clojure_test_facade.clj

```diff
@@ -30,7 +30,7 @@
                             [])]
     (binding [ct/*test-out* (java.io.StringWriter.)]
       (assoc (apply ct/run-tests namespaces-to-run)
-             :lines (-> ct/*test-out* .toString str/split-lines)))))
+             :lines (->> ct/*test-out* .toString str/split-lines (remove empty?))))))
```

Comment:
I eventually reverted these changes in https://github.com/marick/Midje/pull/394/commits/617fadad509988fff190149e21bc580e4edf54a0 because I couldn't get changes to pass on both my machine and travis.